libs/blocks/hero-marquee/hero-marquee.css

```diff
@@ -0,0 +1,554 @@
+.hero-marquee .norm .action-area { margin-top: var(--spacing-s); }
+.hero-marquee .norm div > *:last-child { margin-bottom: 0; }
+.hero-marquee .norm div *:first-child { margin-top: 0; }
+
+.hero-marquee > .foreground {
+  max-width: var(--grid-container-width);
+  min-width: var(--grid-container-width);
+  margin: 0 auto;
+  display: grid;
+  gap: var(--spacing-m);
+}
+
+html[dir="rtl"] .hero-marquee .foreground {
+  flex-direction: row-reverse;
+}
+
+.hero-marquee > .foreground.fw {
+  width: 100%;
+}
+
+.hero-marquee .foreground .copy {
+  display: grid;
+  gap: var(--spacing-xs);
+}
+
+.hero-marquee hr {
+  width: 100%;
+  border: none;
+  height: 1px;
+  margin: 0;
+}
+
+.hero-marquee .has-divider {
+  margin: var(--spacing-xxs) 0;
+}
+
+/* content hidden */
+.hero-marquee .hidden {
+  visibility: hidden;
+  height: 0;
+}
+
+.hero-marquee .foreground div:empty {
+  display: none;
+}
+
+/* Row Types */
+.hero-marquee .row-list {
+  text-align: left;
+} 
+
+html[dir="rtl"] .hero-marquee .row-list {
+  text-align: right;
+}
+
+.hero-marquee .row-list .row-wrapper {
+  display: table;
+}
+
+.hero-marquee .row-list li {
+  margin-bottom: var(--spacing-xxs);
+} 
+
+/* row-qr */
+.hero-marquee .row-qrcode .row-wrapper {
+  display: flex;
+  gap: var(--spacing-s);
+  margin: 0;
+}
+
+.hero-marquee .row-qrcode .row-wrapper > p {
+  margin: 0;
+}
+
+.hero-marquee .row-qrcode .qr-code-img {
+  display: none;
+}
+
+.hero-marquee .row-qrcode .google-play a,
+.hero-marquee .row-qrcode .app-store a {
+  width: 135px;
+  height: 40px;
+  display: inline-flex;
+  color: transparent;
+}
+
+.hero-marquee .row-qrcode .google-play a {
+  background: url('/libs/img/ui/google-play.svg') no-repeat transparent;
+}
+
+.hero-marquee .row-qrcode .app-store a {
+  background: url('/libs/img/ui/app-store.svg') no-repeat transparent;
+}
+
+.hero-marquee.center .row-qrcode .row-wrapper {
+  justify-content: center;
+}
+
+.hero-marquee.center .row-list .row-wrapper { 
+  margin: 0 auto; 
+}
+
+.hero-marquee .row-list .icon-list {
+  padding-inline-start: var(--spacing-m);
+  margin: 0;
+}
+
+.hero-marquee li.icon-item {
+  position: relative;
+  list-style: none;
+}
+
+.hero-marquee li::marker {
+  padding-inline-start: 40px;
+}
+
+.hero-marquee li.icon-item::marker {
+  color: transparent;
+}
+
+.hero-marquee li.icon-item span.icon {
+  position: absolute;
+  left: -28px;
+}
+
+html[dir="rtl"] .hero-marquee li.icon-item span.icon {
+  left: unset;
+  right: -28px;
+}
+
+/* row-supplemental */
+.hero-marquee .row-supplemental.bold {
+  font-weight: 700;
+}
+
+.hero-marquee.fg-media-cover.has-bg .asset {
+  display: none;
+}
+
+.hero-marquee.asset-left > .foreground.cols-2 > .asset {
+  order: 2;
+}
+
+/* mobile ONLY */
+@media (max-width: 600px) {
+  .hero-marquee .con-button {
+    display: block;
+    text-align: center;
+  }
+  
+  .hero-marquee.fg-media-stack-top-mobile,
+  .hero-marquee.fg-media-stack-top-mobile-tablet {
```

Comment:
The discussion going on [here](https://github.com/orgs/adobecom/discussions/2420#discussioncomment-9692041) made me think of this.

If there is a need to differentiate between background and foreground images, I think we should default to one and call out the other.

- `media`
- `bg-media`

Seeing `fg` threw my brain for a loop. I had no idea what that was. Seeing `bg` is more recognizable as background. Maybe that's just me? Any other opinions?

I do think adding `stack` is overly verbose. I think the position name says it all. 

- `media-top-mobile`
-  `bg-media-top-mobile` 

 tell me what I need to know about those options.
 
 I also think `mobile-tablet` in `fg-media-stack-top-mobile-tablet` reads odd. 
 Is this an effort to combine the two? 
 
 I think the more common pattern in use is to declare both. `media-top-mobile` AND `media-top-tablet`

Reply:
+1 for `bg` over `fg`. We're already using `bg` in [styles.css](https://github.com/adobecom/milo/blob/cde7824f4dc0e07b041dfebc03efa33f5e6523cc/libs/styles/styles.css#L453)

Reply:
...eating my words because apparently `fg` is also used in [home.css](https://github.com/adobecom/milo/blob/cde7824f4dc0e07b041dfebc03efa33f5e6523cc/libs/blocks/home/home.css#L45). Still though, bg does seem like much more common of a convention than fg.

Reply:
Wait, [home.css](https://github.com/adobecom/milo/blob/cde7824f4dc0e07b041dfebc03efa33f5e6523cc/libs/blocks/home/home.css#L45).? What does the home block in Milo do? Not sure that should be an guiding example.

Reply:
I too have this question, haha. Maybe some analytics could tell us if it's even being used anywhere?

Reply:
@rgclayton && @elan-tbx Thanks for the thoughts on these things. After thinking thru these patterns, I am going to drop all prefixed naming conventions of `fg-` (foreground) and it's implied that media is targeting this content type. I am also going to omit the `stack-` part of these variants as `top` or `bottom` is enough to distinguish. 
I am going to keep the `bg-` prefix as thats pretty universally known to represent background. I'll comment on this task once that change is made and reflected in this feature branch and supporting drafts.

Reply:
✅ This branch and ref. docs have been updated.